openmetadata-ui/src/main/resources/ui/src/components/common/TestConnection/TestConnection.test.tsx

```diff
@@ -331,6 +331,9 @@ describe('Test Connection Component', () => {
     expect(
       screen.getByText('message.test-connection-taking-too-long')
     ).toBeInTheDocument();
+
+    // 59 since it will make this amount of call, and after timeout it should not make more api calls
+    expect(getWorkflowById).toHaveBeenCalledTimes(59);
```

Comment:
i think we should also add negative assertion here
expect(getWorkflowById).not.toHaveBeenCalledTimes(60);

Reply:
updated!